### ISSUE TITLE
Non-additive feature behavior

### ISSUE THREAD

**Issue by dtolnay** [nvzqz/static-assertions-rs#13]
This code works:

```rust
use static_assertions::const_assert;

fn main() {
    const_assert!(false != true);
}
```

until any other crate in my dependency graph enables static_assertions' "nightly" feature. Then it breaks my crate's formerly working code. This is not good behavior for a cargo feature. Code that works without a feature must also work with the feature, otherwise you end up with libraries that work in isolation but can't be combined together in one dependency graph.

**Comment by nvzqz** [comment]
🤦‍♂️ wow I had never considered this, thank you for letting me know. I suppose the only real solution is to remove the `const _` bit until it becomes stabilized in 1.37 (https://github.com/rust-lang/rust/pull/61347).

**Comment by nvzqz** [comment]
Closed with the release of 1.0. 😄
